prometheus-exporters-formula/prometheus-exporters/init.sls

```diff
@@ -20,6 +27,13 @@ exporter_exporter:
       - pkg: exporter_exporter
     - watch_in:
       - service: exporter_exporter
+    - names:
+      - {{ exporters.exporter_exporter_service_config }}:
+        - source: {{ 'salt://prometheus-exporters/files/exporter-exporter-config.' ~ salt['grains.get']('os_family') }}
+  {% if grains['os_family'] == 'Debian' %}
+      - /etc/exporter_exporter.yaml:
+        - source: salt://prometheus-exporters/files/exporter-exporter.yaml
```

Comment:
What is this file used for, and why is it only needed for Debians?

Reply:
It's the central empty configuration file in [directory-based configuration setup](https://github.com/QubitProducts/exporter_exporter#directory-based-configuration). In SUSE we have it in [the package](https://build.opensuse.org/package/view_file/server:monitoring/golang-github-QubitProducts-exporter_exporter/exporter_exporter.yaml?expand=1). Debian upstream package does not include it. Instead it provides [configuration](https://salsa.debian.org/go-team/packages/prometheus-exporter-exporter/-/blob/debian/sid/expexp.yaml) for exporters we don't ship, so it has to be reconfigured anyway.